pytest_splunk_addon/standard_lib/event_ingestors/hec_raw_ingestor.py

```diff
@@ -61,10 +61,19 @@ def ingest(self, events):
         main_event = []
         param_list = []
         for event in events:
+            if event.metadata.get("host_type") in ("plugin", None):
```

Comment:
get returns `None` if key doesn't exist.  And if key doesn't exist, `event.metadata["host"]` will be an exception

Reply:
If the host_type is None it will take host from event metadata.